README.md

```diff
@@ -24,24 +28,31 @@ Linux kernel module for adding GPIO bitbanged I²C host device.
 
 It accepts following parameters:
   - `busid` (default 7) - I²C bus id (will create i2c-`busid` device)
-  - `sda` (default 0) - GPIO pin number to be used as `SDA`
-  - `scl` (default 1) - GPIO pin number to be used as `SCL`
-  - `udelay` - SCL frequency is (500 / `udelay`)
-  - `timeout` - clock streaching timeout
-  - `sda_od` (bool) - SDA is configured as open drain
-  - `scl_od` (bool) - SCL is configured as open drain
-  - `scl_oo` (bool) - SCL output drivers cannot be turned of (no clock stretching)
+  - `sda` (int, default 0) - GPIO pin number to be used as `SDA`
+  - `scl` (int, default 1) - GPIO pin number to be used as `SCL`
+  - `udelay` - (int, default 50 for scl_oo==1 or 5 otherwise) SCL frequency is (500KHz / `udelay`)
+  - `timeout` - (int, default 100) clock stretching timeout in milliseconds 
+  - `sda_od` (bool, default 0) - SDA is configured as open drain*
+  - `scl_od` (bool, default 0) - SCL is configured as open drain*
+  - `scl_oo` (bool, default 0) - SCL output drivers cannot be turned of (no clock stretching)
+
+*Note that true open drain mode is not supported on RaspberyPi so please only use sxx_od=0 for open drain emulation
```

Comment:
I don't really like this comment as it's not general (focuses on RaspberryPi). Most of the boards/SoCs also do not support open drain mode directly but you can still use external logic to enable that. Some more general description would be better, possibly with a link to an "Open Drain and Open Source" section of Documentation/gpio/gpio.txt.